.env

```diff
@@ -0,0 +1,8 @@
+API_INTERNAL_PORT=8000
+API_PORT=9000
+
+WEB_PORT=10589
+
+MINIO_DEFAULT_BUCKET=sample
+MINIO_ROOT_USER=root
+MINIO_ROOT_PASSWORD=12345678
```

Comment:
maybe add 9 to make it more secure

Reply:
it has an 8 char min, normally i used 1234 for maximum security